src/Utility/AnaSol.h

```diff
@@ -46,24 +47,25 @@ class AnaSol
   //! allocated objects, as the class destructor will attempt to delete them.
   AnaSol(RealFunc* s1 = 0, VecFunc* s2 = 0,
          VecFunc* v1 = 0, TensorFunc* v2 = 0, STensorFunc* v3 = 0)
-    : scalSol(s1), scalSecSol(s2), vecSol(v1), vecSecSol(v2), stressSol(v3) {}
+    : scalSol{s1}, scalSecSol{s2}, vecSol(v1),
+      vecSecSol(v2), stressSol(v3) {}
```

Comment:
Isn't it better (at least more clear) with the following body, to avoid null-pointers in the array of scalar solutions:
```
{
  if (s1) scalSol.push_back(s1);
  if (s2) scalSecSol.push_back(s2);
}
```
And if so, put it in the C-file instead. Actually, think you need to do it this way, or else the hasScalarSol method below will not yield correct result since it does not check for null pointers in the array.